Word Template.java

```diff
@@ -26,12 +30,15 @@ public static String scrambleWord(String word)
      *    before the method was called
      */
     public static void scrambleOrRemove(List<String> wordList){
-        /* part (b) */
-
+     for(i=0;i<wordList.size;i++){ 
+      String s= scrambleWord(wordList.get(i));
+      if(wordList.get(i).equals(s)){
+           wordList.remove(i);
+      }
```

Comment:
When you removed one element from an ArrayList, you must add i-- to avoid the next element to be skipped。